scripts/cancel_all_but.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Hmm. So, we need gh binary for this to work right?

Reply:
Yes, locally.
You also need to be logged in on Github via `gh cli`